src/pages/DirectMessage.tsx

```diff
@@ -201,9 +251,40 @@ const DirectMessage = () => {
               <InfoOutlinedIcon />
             </IconButton>
           </Box>
-          <Divider />
+
           <Box sx={styles.chatContainer}>
             <List component="div" ref={messageRef} sx={styles.messageList}>
+              <Box
+                sx={styles.bioContainer}
+                onClick={() => navigate(`/${selectedConversation.username}`)}
+              >
+                {loading ? (
+                  <CircularProgress />
+                ) : (
+                  <>
+                    <Box sx={styles.bioDetails}>
+                      <Avatar sx={styles.avatar} />
+                    </Box>
+                    <Box sx={styles.nameContainer}>
+                      <Typography variant="h6" sx={styles.displayName}>
+                        {selectedConversation.displayName}
+                      </Typography>
+                      <Typography variant="body2">
+                        {`@${selectedConversation.username}`}
+                      </Typography>
+                    </Box>
+                    <Typography>{selectedConversation.bio}</Typography>
```

Comment:
Can we make this conditional since not everyone has a bio? I'm not sure if it affects any styling but just in case

Reply:
Done